fastani/1.34-RGDV2/Dockerfile

```diff
@@ -0,0 +1,76 @@
+## build RGDv2 ##
+FROM staphb/ncbi-datasets:latest as stage
```

Comment:
I know ncbi-datasets changes a lot, but it's good practice to use a specific tag instead of latest. Could you change staphb/ncbi-datasets:latest to something like staphb/ncbi-datasets:15.11.0

Reply:
You're right! I am going to change it.